source/_integrations/renson.markdown

```diff
@@ -17,11 +17,58 @@ ha_platforms:
   - binary_sensor
   - button
   - sensor
+  - fan
   - number
   - fan
 ha_integration_type: integration
 ---
 
-The Renson integration pulls in data from the Renson Endura delta device. Most of the sensors that can be monitored from inside the Android/iOS application can be monitored with this integration.
+The Renson integration pulls in data from the Renson Endura delta application for Android en iOS. It also provides some services to control the ventilation unit.
+
+{% include includes/config_flow.md %}
```

Comment:
Could you tell me what you're trying to include here?

Reply:
It was an include that came from a wrong git rebase action. Fixed that